src/pip/_internal/commands/download.py

```diff
@@ -76,6 +78,9 @@ def add_options(self) -> None:
 
     @with_cleanup
     def run(self, options: Values, args: List[str]) -> int:
+        if options.parallel_downloads < 1:
+            raise CommandError("Value of '--parallel-downloads' must be greater than 0")
```

Comment:
Could this be use friendly, by instead of raising CommandError, surfacing the error to the user, but defaulting to 1, even here?

Reply:
Do you mean raise an error if the user sets `--parallel_downloads` to 0, then set it to 1 and continue the install/download operation? 
That might be more user friendly but I think it'd be better to be more explicit and throw an error with a clear message.